File: w06_cs/Geometry/Cylinder.cs
using System;

namespace Geometry
{
    public class Cylinder
    {
        private double radius;
        private double length;

        public Cylinder(double radius, double length)
        {
            this.radius = radius;
            this.length = length;
        }

        public double Area()
        {
            return 2 * Math.PI * radius * (radius + length);
        }

        public double Volume()
        {
            return Math.PI * radius * radius * length;
        }

        public void PrintInfo()
        {
            Console.WriteLine($"Cylinder radius={radius}, length={length}, area={Area()}, volume={Volume()}");
        }
    }
}

File: w06_java/src/geometry/Cylinder.java
package geometry;

public class Cylinder {
    private double radius;
    private double length;

    public Cylinder(double radius, double length) {
        this.radius = radius;
        this.length = length;
    }

    public double area() {
        return 2 * Math.PI * radius * (radius + length);
    }

    public double volume() {
        return Math.PI * radius * radius * length;
    }

    public void printInfo() {
        System.out.println("Cylinder radius=" + radius + ", length=" + length +
                           ", area=" + area() + ", volume=" + volume());
    }
}
